README.md

```diff
@@ -6,128 +6,44 @@ This is a client side library that implements
-The following is an example of adding a client filter to a JAX-RS based framework that creates a dropwizard/jersey client.
-
-Client instrumentation example:
-```java
-import com.uber.jaeger.dropwizard.TracingUtils;
-
-// JaegerConfig gets initialized by whichever dropwizard configuration reader you are using.
-com.uber.jaeger.dropwizard.Configuration jaegerConfig;
-
-final Client client = new JerseyClientBuilder(env)
-    /* Arbitrary configuration code … */
-    .register(TracingUtils.clientFilter(jaegerConfig);
-```
-
-Your server configuration will probably not look exactly like the one below.  However JAX-RS server configurations always expose a `register` function that can be used to register a server filter like below.
-Server instrumentation example:
-```java
-Final ResourceConfig rc = new ResourceConfig()
-    /* Arbitrary configuration code… */
-    .register(TracingUtils.serverFilter(jaegerConfig));
-
-return GrizzlyHttpServerFactory.createHttpServer(URI.create(BASE_URI), rc);
-```
-
-## Apache http client instrumentation ##
-See Apache http client's [README.md](./jaeger-apachehttpclient/README.md)
-
-
-### Metrics ###
-In order for Jaeger to set metrics properly it needs to be configured with a metric registry.  
-```java
-import com.uber.jaeger.dropwizard.TracingUtils;
-
-// Dropwizard's metrics registry class
-MetricsRegistry registry = new MetricsRegistry();
-
-// JaegerConfig gets initialized by whichever dropwizard configuration reader you are using.
-com.uber.jaeger.dropwizard.Configuration jaegerConfig;
-
-jaegerConfig.setMetricRegistry(registry);
-
-final Client client = new JerseyClientBuilder(env)
-    /* Arbitrary configuration code … */
-    .register(TracingUtils.clientFilter(jaegerConfig);
-```
-
-## Thread Pooling: ##
-`jaeger-context` defines [ThreadLocalTraceContext](https://github.com/uber/jaeger-client-java/tree/master/jaeger-context/src/main/java/com/uber/jaeger/context) implementation of `TraceContext` that can be used for propagating the current tracing `Span` throughout the application without changing the application's code. However, if the application is starting new threads or is using thread pools, the thread-local context is not going to be carried over into the execution in the next thread. To maintain context propagation, a wrapper `TracedExecutorService` is provided that automatically transfers the context onto the new threads.
+All artifacts are published to Maven Central. 
+Snapshot artifacts are also published to
+[Sonatype](https://oss.sonatype.org/content/repositories/snapshots/com/uber/jaeger/).
+Follow these [instructions](http://stackoverflow.com/questions/7715321/how-to-download-snapshot-version-from-maven-snapshot-repository)
+to add the snapshot repository to your build system. 
+
+Add the required dependencies to your project. Usually, this would only be a single dependency.
+**Please use the latest version:** [![Released Version][maven-img]][maven]
```

Comment:
I would still list at least one example of POM file in the top readme, e.g. for core